13/main/index.html

```diff
@@ -0,0 +1,1480 @@
+</li>
+<li class='spons'>Online&nbsp;&nbsp; Recharge&nbsp;&nbsp; Partner</li>
+<li>
+<img alt='image' src="../Misc/sponsors/images/n_freecharge.jpg" width="150"/>
+</li>
+
+<li class='spons'>Media&nbsp;&nbsp; Partners</li>
+<li>
+<img alt='image' src="../Misc/sponsors/images/p_ennadu.jpg" width="150"/>
+<img alt='image' src="../Misc/sponsors/images/q_ETV.jpg" width="150"/>
+<img alt='image' src="../Misc/sponsors/images/s_jam.jpg" width="150"/>
+</li>
+<li class='spons'>EVENT SPONSORS</li>
+<li>
+<img alt='image' src="../Misc/sponsors/images/v_servo.jpg" width="150"/>
+</li>
+<li class='spons'>Online Promotion&nbsp;&nbsp; Partners</li>
+<li>
+<img alt='image' src="images/sponsors/Tagmuse.jpg" width="150"/>
+</li>
+<li class='spons'>T Shirt&nbsp;&nbsp; Partner</li>
+<li>
+<img alt='image' src="images/sponsors/estyle.jpg" width="150"/>
+</li>
+</ul>
+
+
+<br /><br />
+                To sponsor Felicity contact us asap!<br />
+                felicity [at] students [dot] iiit [dot] ac [dot] in
+                </div></div>
+        </div>
+
+        <div class="unit item oneWide oneHigh sponsors info about" id="typography">
+        <h4 class="infoHeader">Prev-Sponsors</h4>
+                <div class="content cf">
+                <div class="column-three last">
+                <h1>Past Sponsors</h1>
+                <p class="tagline">We value our sponsors as much as we value our fest.</p>
+                <div class="sponsor"><img alt='image' class="sponsor" src="images/ps.jpg"/></div>
+<br /><br />
+                </div></div>
+        </div>
+
+
+<!-- End Spons -->
+
+
+
+
+	<div class="unit item oneWide oneHigh  nights isotope-item" id="nights_inaugral1">
+                <img alt='image' class="backdrop" src="images/pro_thumb.jpg"  />
+		<h4 class="infoHeader"  >Rock Night</h4>
+
+		<div class="content cf">
+			<div class="column-three last">
+			</div>
+			<div class="column-two">
+                        <br /><br/><br />
+				<p>
+ <img alt='image' src="images/rock.jpg" />
+
+                                </p>
+                                <p>
+A last day event to end on a high note. Be there and experience music in the form it should be in.
+<br /> Visiting band : Scribe Core. Visit Schedule tab for more.<br />
+<br /><br /> Date : 23 February 2013
+                                </p>
+
+
+
+			</div>
+
+		</div>
+	</div>
+
+
+
+
+
+
+
+	<div class="unit item oneWide oneHigh  illustration isotope-item" id="nights_rock">
+                <img alt='image' class="backdrop" src="images/detailed.jpg"  />
+		<h4 class="infoHeader"  >Detailed Schedule</h4>
+
+		<div class="content cf">
+			<div class="column-three last">
+			</div>
+			<div class="column-two">
+                        <br /><br/><br />
+				<p>
+<br /><br /><br /><br />
+<a href="http://10.4.3.65/13/Misc/images/pass_new_back.jpg" target="_blank"> Click here for a detailed schedule </a>
```

Comment:
We really don't want any IPs turning up in this version. I didn't find the image in a quick search, so I'm okay with just editing the URL to make it relative and letting it 404.